packages/connect/src/protocol/invoke-implementation.ts

```diff
@@ -106,7 +106,7 @@ export function transformInvokeImplementation<
             const output = normalizeIterable(
               spec.method.output,
               spec.impl(
-                await ensureSingle(input, "server-streaming"),
+                await ensureSingle(req.message, "server-streaming"),
```

Comment:
Makes sense, but we should double check with https://github.com/connectrpc/examples-es/pull/2452 before cutting a release.

Reply:
Checked against the example by overriding the connect package: https://github.com/connectrpc/examples-es/compare/Add-example-for-interceptors...sk/interceptor-check?expand=1

Reply:
Thank you for confirming!